semcore/radio/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [6.19.1] - 2024-01-24
+
+### Added
+
+- Specific css selector for `Text` component.
```

Comment:
I would write something like "**Changed** css selector specify to prevent rare edge case bug"